artifacts/definitions.py

```diff
@@ -40,7 +40,6 @@
     'IM': 'Instant Messaging / Chat applications artifacts.',
     'iOS': 'Artifacts related to iOS devices connected to the system.',
     'History Files': 'History files artifacts e.g. .bash_history.',
-    'KnowledgeBase': 'Artifacts used in knowledge base generation.',
```

Comment:
there's only one of these now, do you still want to remove it? If so I'll have to go and see if the grr code uses it.